frontend/package.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "frontend",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "watch": "chokidar \"src/**/*\" \"next.config.js\" --initial --command=\"npm run watch_build\"",
```

Comment:
Ah, that's because of the way we're loading translations into this repository (independently of this change); presumably you didn't add `--recurse-submodules` when cloning this repository. You can fix that by running `git submodule update --remote` in the root of the repo, and I added a more helpful error message for that in 7efc093dd97ffbb33d7d68cf181c74553c3701d8.